scripts/set-ipaddress.service

```diff
@@ -0,0 +1,14 @@
+#  This file is part of Minishift
```

Comment:
Consumed by Minishift?

Reply:
other system units are marked as "This file is part of systemd" to distinguish the copyright

Reply:
@LalatenduMohanty add the license header here in-place?